static/app/components/tooltip.tsx

```diff
@@ -224,14 +225,6 @@ export function DO_NOT_USE_TOOLTIP({
   }, []);
 
   function renderTrigger(triggerChildren: React.ReactNode, ref: React.Ref<HTMLElement>) {
-    const containerProps: Partial<React.ComponentProps<typeof Container>> = {
-      'aria-describedby': tooltipId,
-      onFocus: handleMouseEnter,
-      onBlur: handleMouseLeave,
-      onPointerEnter: handleMouseEnter,
-      onPointerLeave: handleMouseLeave,
-    };
```

Comment:
Can we just give these a better type? Would be nice to not have duplication of props

Reply:
👍🏼